imaging/ml/ml_codelab/breast_density_cloud_ml.ipynb

```diff
@@ -822,9 +805,10 @@
    },
    "outputs": [],
    "source": [
-    "# DICOM study and series UID of input mammography image that we'll push for inference.\n",
+    "# DICOM Study/Series UID of input mammography image that we'll push for inference.\n",
     "input_mammo_study_uid = \"1.3.6.1.4.1.9590.100.1.2.85935434310203356712688695661986996009\"\n",
-    "input_mammo_series_uid = \"1.3.6.1.4.1.9590.100.1.2.374115997511889073021386151921807063992\""
+    "input_mammo_series_uid = \"1.3.6.1.4.1.9590.100.1.2.374115997511889073021386151921807063992\"\n",
+    "input_mammo_instance_uid = \"1.3.6.1.4.1.9590.100.1.2.289923739312470966435676008311959891294\""
```

Comment:
Is this another change related to switching to the dataset hosted in GCS. If so - please revert, and lets use the dataset hosted in CHC API (and do it in a separate PR).

Reply:
https://github.com/GoogleCloudPlatform/healthcare/issues/433